orc8r/gateway/python/magma/magmad/bootstrap_manager.py

```diff
@@ -371,7 +371,7 @@ def _is_valid_certificate(self, cert):
         not_before = cert.not_before.ToDatetime()
         if now < not_before:
             logging.error(
-                'Received a not-yet-valid certificate from: %s', not_before)
+                'Cert notBefore time occurs after the current system time; check ntp? notBefore time is %s', not_before)
```

Comment:
`'Current system time indicates certificate received from %s is not yet valid. Consider checking NTP.'`

Reply:
thanks for the rephrase. this reads better :) will update.